css/styles.css

```diff
@@ -127,6 +178,10 @@ i {
   #entry-form-tag {
     margin-left: 22rem;
   }
+
+  .save-container {
+    padding-left: 35.5rem;
```

Comment:
You should avoid using such large rem values, you should really only be going up to about `5rem`. If you have such large values, then you're probably doing something "hacky" to get this to work